src/app/sb-admin/components/pages/sb-user/add-edit-user/add-edit-user.component.html

```diff
@@ -0,0 +1,85 @@
+<p-toast></p-toast>
+<form [formGroup]="addEditUserForm">
+    <div class="card p-fluid">
+        <div class="field">
+            <label for="firstName">First Name * </label>
+            <input type="text" pInputText id="firstName" autofocus
+                [ngClass]="{'ng-invalid ng-dirty' : submitted && addEditUserForm.controls['firstName'].errors}"
+                formControlName="firstName" />
+            <div>
+                <small *ngIf="submitted && addEditUserForm.controls['firstName'].errors">First Name is required.</small>
+            </div>
+        </div>
+        <div class="field">
+            <label for="lastName">Last Name *</label>
```

Comment:
remove this hardcoded label and use language constant

Reply:
resolved